packages/orbit-components/src/primitives/BadgePrimitive/__tests__/index.test.tsx

```diff
@@ -8,21 +8,16 @@ describe("BadgePrimitive", () => {
   it("should have expected DOM output", () => {
     render(
       <BadgePrimitive
+        className="orbit-kek"
         icon={<Sightseeing dataTest="icon" />}
-        background="red"
-        foregroundColor="blue"
         dataTest="test"
         ariaLabel="label"
       >
         badge
       </BadgePrimitive>,
     );
 
-    expect(screen.getByTestId("test")).toHaveStyle({
-      backgroundColor: "red",
-      color: "blue",
-    });
-    expect(screen.getByLabelText("label")).toBeInTheDocument();
```

Comment:
Because does not make sense to test it by label, if it's already [tested here ](https://github.com/kiwicom/orbit/pull/4031/files#diff-c7dd1eecef03463652c88e8ae17dd9db322641349e303f3dfc00254a399c14de)